tests/conftest.py

```diff
@@ -90,6 +93,9 @@ def pytest_collection_modifyitems(config, items):
                 if "no_parallel" in item.keywords or not also_run_parallel:
                     continue
 
+            if "fake_gcp" in item.keywords:
```

Comment:
Why skip fake gcp tests for parallel execution?

Reply:
In "parallel execution", bionic executes parts of the flow in a separate process. The fakes / mocks do not work in a separate process.

Reply:
Would it work if we use dependency injection instead of global mocks? If it does, I think it would be worth investing in it now. @jqmp wdyt?

Reply:
Hmm, that is a good point. But I am thinking of investigating FSSPEC first before looking into refactoring all the GCS calls to make it injectable.

Reply:
I agree that we should either use FSSPEC (if feasible) or use injection (if it will make the parallel tests work). I don't feel strongly about which one we should try first.